04_lists-and-attributes.Rmd

```diff
@@ -1,13 +1,346 @@
-# Lists and attributes
+# Lists and Attributes
 
-**Learning objectives:**
+## Converting between Vector Types 
```

Comment:
I recommend adding ` {-}` to all of the `##`-level headings (and below), so they don't get weird/incorrect numbers. Some day I'll sort out how to fix this globally, but until then it looks best if you do it manually.

```suggestion
## Converting between Vector Types {-}
```